deploy/push_to_rdss_willow.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+cd /home/willow/willow
+git push -u rdss-willow master
```

Comment:
Actually ... can we have codeship do this rather than the demo server? Jisc would just need to add our project's codeship public key, then as part of each build setup, codeship pushes to their repo. That way we have no scripts, no nothing to maintain on the demo server, which simply continues following our core Willow repo.